content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,250 @@
+entered here include :guilabel:`Meal Vouchers`, :guilabel:`Fuel Card`, :guilabel:`Internet`,
+:guilabel:`Paid Time Off`, etc.
+
+Some fields may be automatically filled in based off of the contracts selected in the
+:guilabel:`Contract Details` tab.
+
+Attachment of salary
+--------------------
+
+.. image:: contracts/salary-attachment.png
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+Any automatic deductions or allocations for an employee, such as child support payments and wage
+garnishments, are referred to as a *salary attachment*. This section is where all of these
+deductions or allocations are set.
+
+To add a new deduction, click :guilabel:`Add a line`. Type in a description for the allocation under
+:guilabel:`Description`.
+
+.. image:: contracts/garnishment.png
+   :align: center
+   :alt: Enter a new line for each type of garnishment.
+
+Select the :guilabel:`Garnished Type` from the drop-down menu. Choose from:
+
+- :guilabel:`Attachment of Salary`: Any payments taken out towards something that is *not* child
+  support. Typically any garnishments such as lawsuit payments, payments toward taxes owed, etc.
+- :guilabel:`Assignment of Salary`: Any deduction that is not required but voluntary, such as a
+  pre-tax allocation to a college savings account.
+- :guilabel:`Child Support`: Any payments taken out specifically for child support.
+
+Enter the start and end dates the entry applies to. Click on the drop-down menu under
+:guilabel:`From` and :guilabel:`To`, navigate to the correct month and year by using the arrow
+icons, then click on the date.
+
+Last, enter the :guilabel:`amount` that each payslip pays towards the entry.
+
+To delete a line, click the :guilabel:`🗑️ (Trash Can)` icon at the end of the line.
+
+.. image:: contracts/delete-garnishment.png
+   :align: center
+   :alt: Remove a new line by clicking the delete icon.
+
+Save and send the contract
+--------------------------
+
+Once a contract has been created and/or modified, save the contract by clicking the :guilabel:`Save`
+button. Next, the contract must be sent to the employee to be signed.
+
+Click on one of the following buttons to send the contract to the employee:
+
+.. image:: contracts/send-contract.png
+   :align: center
+   :alt: Send the contract to the employee via one of the buttons.
+
+- :guilabel:`Generate Simulation Link`: This option is for Belgian companies only. Clicking this
+  opens a pop-up window that contains the basic information from the contract as well as a link for
+  the contract when using the salary configurator. Click :guilabel:`Send Offer` to send an email to
+  the employee so they can sign the contract.
+
+.. image:: contracts/simulation.png
+   :align: center
+   :alt: Sends a link to the employee for the contract.
+
+.. note::
+   In order to send a contract using the :guilabel:`Generate Simulation Link`, there must be a
+   signature field in the contract PDF being sent to the employee so they can sign it.
```

Comment:
I think you may want to indent these two elements to be within the bullet point above, since the image and note are specific to that point